DEPLOYMENT_PROCEDURE.md

```diff
@@ -4,23 +4,26 @@ This document is only intended for Pathoplexus sysadmins.
 
 ## To develop new versions of Pathoplexus
 
-Work in a development branch. You can choose what version of Loculus to target by adjusting the `loculusVersion` in  `pathoplexus_app/values.yaml`. Use a Loculus commit SHA.
+Work in a development branch. You can choose what version of Loculus to target by adjusting the `loculusVersion` in  `pathoplexus_app/values.yaml` (use a Loculus commit SHA). Adding the `preview` label will create a preview.
 
-## Merging to main
+## To simply bump the version of Loculus used in Pathoplexus `main`
+We have a GitHub workflow for this. Click Actions -> Create PR to bump main pathoplexus.. -> Run workflow -> Run workflow. This will create a PR, which someone else will need to approve. The PR will contain a changelist for Loculus, which you should carefully review for breaking changes.
 
-Open a PR to merge into main. The current main is visible at http://preview-main.pathoplexus.org.
+## Staging the latest main
 
-## Staging a new version
+Once you have new release candidate on main, you can point the staging server at it by changing the head_sha in the `staging` folder in https://github.com/pathoplexus/loculus_deployments/tree/main/deploy .
```

Comment:
Should we instead directly link to the file https://github.com/pathoplexus/loculus_deployments/blob/main/deploy/staging/config.json?